src/lang/lang_types.ml

```diff
@@ -492,8 +486,10 @@ let print_repr f t =
         let vars = print ~par:false vars t in
         Format.fprintf f "]@]";
         vars
-    | `Getter a ->
-        Format.fprintf f "{%s}" (print_ground a);
+    | `Getter t ->
+        Format.fprintf f "{";
+        let vars = print ~par:false vars t in
+        Format.fprintf f "}";
```

Comment:
Are we concerned that this will clash with the syntax `{v}` for `()->v`?

Reply:
It's not worse than earlier and the two are "mostly consistent".